### DISCUSSION TITLE
Searching for "Sparks" in "Any" category in beatmap listing overlay fails

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#16363] [Q&A] [answered]
If the chosen category is `Any`, typing "Sparks" into the search will fail to complete. `Has leaderboard` category completes the search just fine.

https://user-images.githubusercontent.com/53346335/148647448-b49ed47c-b34f-4de9-a2a9-05fea496f77c.mp4

[network.log](https://github.com/ppy/osu/files/7833352/network.log) (error is on line 183)
[runtime.log](https://github.com/ppy/osu/files/7833353/runtime.log)

**Comment by frenzibyte** [comment, marked as answer]
#16364
